src/control/lib/control/pool_test.go

```diff
@@ -704,9 +702,6 @@ func TestControl_PoolCreate(t *testing.T) {
 			ctx := test.Context(t)
 			mi := NewMockInvoker(log, mic)
 
-			if tc.req.userExt == nil {
-				tc.req.userExt = mockExt
-			}
```

Comment:
Huh, so the test wasn't actually using the mocks for anything?

Reply:
looks like it was until this change https://github.com/daos-stack/daos/pull/14381/files#diff-b86b7f9f25785e10ce878665ccc27f1a331e16ab733a0ff7729b7e404d0d659fR53